controlpanel/frontend/forms.py

```diff
@@ -176,20 +193,64 @@ def __init__(self, *args, **kwargs):
 
 class CreateParameterForm(forms.Form):
     key = forms.CharField(
-        validators=[RegexValidator(r'[a-zA-Z0-9_]{1,50}')]
+        max_length=50,
+        validators=[
+            RegexValidator(
+                r'[a-zA-Z0-9_]{1,50}',
+                message="Must contain only alphanumeric characters and underscores",
+            ),
+        ],
     )
     role_name = forms.CharField(
-        validators=[RegexValidator(r'[a-zA-Z0-9_-]{1,60}')]
+        max_length=60,
+        validators=[
+            RegexValidator(
+                r'[a-zA-Z0-9_-]{1,60}',
+                message="Must contain only alphanumeric characters, underscores and hyphens",
```

Comment:
As said for the other validator, this is nicer. I'd mentioned the character limit as well here.